src/screens/internal/AllStates/AllStates.tsx

```diff
@@ -3,22 +3,26 @@ import { LoadingScreen } from './AllStates.style';
 import { useProjectionsFromRegion } from 'common/utils/model';
 import { Metric } from 'common/metricEnum';
 import { MetricChart } from 'components/Charts';
-import regions from 'common/regions';
+import regions, { State as StateType } from 'common/regions';
 
 function AllStates() {
-  return regions.states.map(state => {
-    return <State key={state.stateCode} state={state} />;
-  });
+  return (
+    <>
```

Comment:
the context in which AllStates() was used required a single child, not a list, so I wrapped in a <>